File: Q15/Q15.js
function printHollowRhombus(n) {
    let result = "";
    for (let i = 1; i <= n; i++) {
        result += " ".repeat(n - i);
        for (let j = 1; j <= n; j++) {
            if (i === 1 || i === n || j === 1 || j === n) {
                result += "*";
            } else {
                result += " ";
            }
        }
        result += "\n";
    }
    console.log(result);
}

let n = 5;
printHollowRhombus(n);

File: Q15/Q15.cpp
#include <iostream>
using namespace std;

void printHollowRhombus(int n) {
    for (int i = 1; i <= n; i++) {
        for (int j = 1; j <= n - i; j++) {
            cout << " ";
        }
        for (int j = 1; j <= n; j++) {
            if (i == 1 || i == n || j == 1 || j == n) {
                cout << "*";
            } else {
                cout << " ";
            }
        }
        cout << endl;
    }
}

int main() {
    int n = 5;
    printHollowRhombus(n);
    return 0;
}
